libraries/c_sdk/standard/https/test/unit/iot_tests_https_client.c

```diff
@@ -324,31 +325,31 @@ TEST_TEAR_DOWN( HTTPS_Client_Unit_API )
  */
 TEST_GROUP_RUNNER( HTTPS_Client_Unit_API )
 {
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ConnectInvalidParameters );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ConnectFailure );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ConnectSuccess );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, DisconnectInvalidParameters );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, DisconnectFailure );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, DisconnectSuccess );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, InitializeRequestInvalidParameters );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, InitializeRequestFormatCheck );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, AddHeaderInvalidParameters );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, AddHeaderFormatCheck );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, AddHeaderMultipleHeaders );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ReadHeaderInvalidParameters );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ReadHeaderVaryingValues );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ReadContentLengthInvalidParameters );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ReadContentLengthSuccess );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ReadContentLengthNotFound );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ReadResponseStatusInvalidParameters );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ReadResponseStatusSuccess );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, WriteRequestBodyInvalidParameters );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, WriteRequestBodySuccess );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, WriteRequestBodyNetworkSendFailure );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ReadResponseBodyInvalidParameters );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ReadResponseBodyNetworkReceiveFailure );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ReadResponseBodyParsingFailure );
-    RUN_TEST_CASE( HTTPS_Client_Unit_API, ReadResponseBodySuccess );
+	RUN_TEST_CASE( HTTPS_Client_Unit_API, ConnectInvalidParameters );
```

Comment:
I don't think we want tabs here?

Reply:
It was an accidental formatting change with the `libraries/freertos_plus/standard/freertos_plus_tcp/uncrustify.cfg` file

Reply:
Reverted the formatting changes